playground/csp/index.html

```diff
@@ -4,10 +4,17 @@
     color: green;
   }
 </style>
+<script nonce="#$NONCE$#">
+  // nonce above verifies that this script is allowed to run with nonce
+  // but also verifies that we don't write two nonces as a result of
+  // setting html.cspNonce in the vite.config.jsg
+  document.querySelector('.inline-js').textContent = 'inline-js: ok'
+</script>
 <script type="module" src="./index.js"></script>
 <p class="linked">direct</p>
 <p class="inline">inline</p>
 <p class="from-js">from-js</p>
 <p class="dynamic">dynamic</p>
 <p class="js">js: error</p>
 <p class="dynamic-js">dynamic-js: error</p>
+<p class="inline-js">inline-js: error</p>
```

Comment:
Would you add the assertion in `vite/playground/csp/__tests__/csp.spec.ts` as well?